### ISSUE TITLE
Implicit payload data type declaration of `getTopics` API

### ISSUE THREAD

**Issue by jhaemin** [payw-org/eodiro-server#54]
@97e57e 

First of all, there is an utility type called [`Record`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt) in TypeScript.

```ts
type Record<K extends string, T> = {
    [P in K]: T;
}
```

Currently `getTopics` returns

```ts
{ [key: string]: string }
```

which can be simply rewritten using `Record`

```ts
Record<string, string>
```

Also the key part should be `TipTopic`, not just generic string type.

```ts
Record<TipTopic, string>
```

**Comment by 97e57e** [comment]
@jhaemin this issue is resolved by #55 !!

**Comment by jhaemin** [comment]
Awesome
